manifest.toml

```diff
@@ -42,10 +44,10 @@ ram.runtime = "50M"
     type = "password"
 
 [resources]
-        [resources.sources.main]
-        url = "https://github.com/sabre-io/Baikal/releases/download/0.9.4/baikal-0.9.4.zip"
-        sha256 = "47a23aa06bc82504e0f216ea81328d9fa00c9c9d9f178d5487ce8960e941e58e"
-        autoupdate.strategy = "latest_github_release"
+    [resources.sources.main]
+    url = "https://github.com/sabre-io/Baikal/archive/refs/tags/0.9.5.tar.gz"
+    sha256 = "4618cf981c49e63a40530c642e0a0575a8a166c11b69279c53dbd53f62f627e6"
+    autoupdate.strategy = "latest_github_release"
```

Comment:
This sounds like two very different files ... The old one (0.9.4) is from the release and has a `vendor/` folder which i assume is from composer build or whatever, and the new one (0.9.5) is only the raw github source and has no vendor/ folder but has a composer.json ... If we want to switch to this, we probably have to call composer ourselves etc ?